samples/pyodide/worker.py

```diff
@@ -1,5 +1,60 @@
-from js import Response
+async def fetch(request):
```

Comment:
Can you move this into a separate samples folder? something like `pyodide_fastapi`. It would be nice to keep the main sample as simple as possible.